src/screens/Authentication/AdminRegister/AdminRegister.tsx

```diff
@@ -62,21 +86,30 @@ function AdminRegisterScreen({
                 name="email"
                 label="email"
                 placeholder=" email@email.com"
-                onChangeText={setEmail}
+                hasError={emailError !== ''}
+                onChangeText={emailInput => handleEmailChange(emailInput)}
               />
               <Text style={styles.smallText}>Create Password </Text>
               <AuthInput
                 name="password"
                 label="password"
                 placeholder=" password"
-                onChangeText={setPassword}
+                hasError={passwordError !== ''}
+                onChangeText={passInput => handlePasswordChange(passInput)}
               />
               <Text style={styles.smallText}>Verify Password </Text>
               <AuthInput
                 name="confirmPassword"
                 label="confirmPassword"
                 placeholder=" password"
+                hasError={passwordError !== ''}
               />
+              {emailError !== '' && (
```

Comment:
could we organize like in AdminSignin.tsx where  you put the emailError !== '' line of code right underneath the email AuthInput